### PR TITLE
More comp test

### PR DESCRIPTION
**Pull request by axelssonHakan**

### CODE REVIEW

File: src/components/filterbox.jsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import ReactDOM from 'react-dom';
 import React, { useRef, useState } from 'react';
 import PropTypes from 'prop-types';
-import Column from 'react-virtualized/dist/es/Table/Column';
+// import Column from 'react-virtualized/dist/es/Table/Column';
```
Comment (hrigner) [inline src/components/filterbox.jsx]: Remove this

File: src/components/hypercube-table.jsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Column from 'react-virtualized/dist/es/Table/Column';
+// import Column from 'react-virtualized/dist/es/Table/Column';
```
Comment (hrigner) [inline src/components/hypercube-table.jsx]: Remove this

File: src/components/virtual-table.jsx [hunk 3]
```diff
@@ -4,7 +4,8 @@ import React, {
 import {
   Table, AutoSizer, InfiniteLoader,
 } from 'react-virtualized';
-import 'react-virtualized/styles.css';
+// import 'react-virtualized/styles.css';
```
Comment (hrigner) [inline src/components/virtual-table.jsx]: Remove this

File: package.json [hunk 4]
```diff
@@ -49,11 +50,13 @@
     "eslint-config-airbnb": "17.1.0",
     "eslint-plugin-import": "2.14.0",
     "eslint-plugin-jsx-a11y": "6.1.2",
+    "jsdom": "^13.0.0",
```
Comment (wennmo) [inline package.json]: lock dependencies

File: test/comp/error-reporting.spec.jsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  });
+
+  it('should display cookie if not set', () => {
+    const sandbox = sinon.createSandbox();
+    sandbox.stub(ReactGA, 'initialize');
+    sandbox.stub(ReactGA, 'pageview');
+    sandbox.stub(window, 'location').value({ pathname: 'some value' });
+    sandbox.stub(process, 'env').value({ GA: 'some value' });
+
+    // process.env.GA = 'string';
```
Comment (wennmo) [inline test/comp/error-reporting.spec.jsx]: remove?

**Pull request merged**